### ISSUE TITLE
Inline-editing of transactions

### ISSUE THREAD

**Issue by justbudget** [justbudget/justbudget#32]
Instead of having to use the "transaction" form, I should be able to edit the fields in the transaction in-line in the accounts view.

<img width="763" alt="Screen Shot 2022-06-03 at 10 30 13 PM" src="https://user-images.githubusercontent.com/106850056/171974768-9a7ffd15-5d1f-42a2-b1a4-7bcd5c9ba5a1.png">

**Comment by bpaulien** [comment]
I would suggest also inline adding of new transactions.  With option to auto start a new transaction, in cases where I have several to enter, it's easier than going back to the mouse and hitting the "+" button between each.  Hitting escape could "cancel" the auto add process.

**Comment by justbudget** [comment]
Yes good point, I'm definitely planning on in-line adding as well. 

Here's a separate issue for adding: https://github.com/justbudget/justbudget/issues/39

**Comment by InsouciantQualms** [comment]
Please be sure to support keyboard only editing as well.  Switching to the mouse and back is a (minor) time sink.

**Comment by justbudget** [comment]
I'll be sure to add a keyboard-only flow to the inline adding/editing, thanks for the suggestion.

There are actually already some keyboard shortcuts for adding transactions, I myself found it cumbersome to keep using the mouse, I'll copy and paste my description from reddit:

On the account view, I hit `q` to start a new transaction. Then `shift` + `←`/`→` to change the date by one day at a time. I hit `tab` to move through the text boxes and then `enter` to create the transaction.

**Comment by InsouciantQualms** [comment]
That's great to know.  Thanks!

**Comment by bpaulien** [comment]
One more comment on this, with another scenario, and use case.  I'm not sure how you want to implement it, but I'd suggest being able to have some way of editing the currently selected record, even if other records are "selected/checked".  I just came across this, where I had already selected a few records, and then realized that I needed to edit one.  So I of course selected it too, but when multiple records are selected, the only options available are delete or mark as cleared.  I wasn't ready to do either, so I had to stop in the middle of my process, deselect everything, and edit the one record.  Now I have to remember where I was in my process, or start over, and hope there's not another edit I have to do while proceeding.

**Comment by bpaulien** [comment]
Oh, that's interesting, I also noticed that if you click on the note field of a record, it won't select/deselect the record.  (also won't do anything else) but if you click anywhere else on the record, it will select/deselect the record.  Not sure if this is intentional, or if there's something special about notes,
